src/modules/jobs/models/pokemon.model.ts

```diff
@@ -0,0 +1,7 @@
+export interface Pokemon {
```

Comment:
I could be wrong, but it seems the way atlas typically does models is as a class which takes in an api response basically verifies it and turns it into a typed variable. This is fine to keep, though, just letting you know but we can ask tyler about it.

Reply:
We can talk about it. I'm coming from the Ruby on Rails background where a model more or less defines a SQL structure, but can be used to define data. Let me know what you would like me to do.

Reply:
Let's just leave it for now 👍